src/controller.cpp

```diff
@@ -555,13 +555,17 @@ int Controller::run(const CliArgsParser& args)
 		std::cout.flush();
 	}
 	try {
-		const std::uint64_t amt = rsscache->cleanup_cache(
+		const auto unreachable_feeds = rsscache->cleanup_cache(
 				feedcontainer.get_all_feeds());
 		if (!args.silent()) {
 			std::cout << _("done.") << std::endl;
-			if (amt > 0u) {
-				std::cout << _("Unreachable feeds found, consider setting "
-						"`cleanup-on-quit yes` or run `newsboat --cleanup`")
+			if (!unreachable_feeds.empty()) {
+				for (const auto& feed : unreachable_feeds) {
+					LOG(Level::USERERROR, "Unreachable feed found: %s", feed);
+				}
+
+				std::cout << strprintf::fmt(_("%d unreachable feeds found. See "
+							"`cleanup-on-quit` in newsboat(1) for details."), unreachable_feeds.size())
```

Comment:
(Hmm, I left a comment here, but it disappear. Reposting.)

This fails to compile on macOS because `strprintf::fmt` doesn't have an overload for the type that `std::vector::size` returns. Please cast it to `std::uint64_t` (should be enough :) and format it with `PRIu64`.

Reply:
I did so.